### DISCUSSION TITLE
Password for Ethernet coordinators?

### DISCUSSION THREAD

**Discussion by mpfl** [tube0013/tube_gateways#141] [Q&A]
I've just (finally) migrated over to the PoE coordinator I bought a year or two ago.

It just struck me that there's no security on the device - anything on the network can connect to it.

Is there any way of adding some security to the coordinator - both the web and serial interfaces?

**Comment by tube0013** [comment]
Unfortunately no. you can create local network firewall rules only allowing your HA or Z2M instance access to the coordinator IP. however there are instances where connecting from another host is useful for firmware update or troubleshooting.
